content/articles/switch-vs-router-vs-hub/index.md

```diff
@@ -0,0 +1,117 @@
+- Hubs operate under half-duplex that's only one occasion of data transmission at a time.
+- Hubs share data to all the devices in a network thus insecure.
+- Hubs does lots of wastage of bandwidth when transmitting data.
+
+### Switch Vs Hub
+Hub is a broadcast device that sends data from one node to all but Switch is a multicast device that can send data to a particular node you want.
+
+### Router
+A router is a networking device that operates under the network layer of the OSI model and is used to connect two or more different networks. It is a device that establishes a common link between networks to set up data flow between them. 
+
+### Advantages of Routers
+- With the aid of dynamic routing algorithms, it has the ability to choose the best path in the internetwork.
+- It creates collision domains to reduce network traffic.
+- It provides connections between network architectures.
+
+### Disadvantages of Routers 
+- They are expensive compared to other devices in a network.
+- They need to analyze data making them slower.
+- Low bandwidth because of dynamic router communication.
+
+Let's look at the differences of each of them:
+
+### Layer of each component in the OSI model
+The [OSI Model](https://www.imperva.com/learn/application-security/osi-model/) (Open Systems Interconnection Model) is a 7 layer model that is used to describe in a pictorial way how computer systems communicate. Switch, Router, and Hub each operate on a different layer.
+
+#### Switch
+A switch is located on the OSI model's Data Link layer i.e the second layer. The link layer is specific to the medium over which the packet is traveling such as Ethernet and the Mac Address are part of this layer.
+
+#### Router
+A router works in the Network Layer of the OSI model and it is the third layer.
+
+#### Hub
+Hub works in the Physical Layer of the OSI model and it is the first layer.
+
+### Function of each Device
+#### Switch
+It allows various connections of many devices in the same network and the management of port and VLAN security settings.
+
+#### Router
+Its major purpose is to connect many types of networks at the same time using adaptive and non-adaptive routing. 
+
+#### hub
+Hub is a simple and cheap networking device that allows a bunch of computers to be connected to a single network 
+
+### Application of each Device
+#### Switch
+It is commonly used in local area networks for connecting many nodes.
+
+#### Souter
+It is commonly used in Local Area Network and Metropolitan Area Network(MAN).
+
+#### hub
+It is similar to Switch because it is used in the local area network(LAN).
+
+### Mode of data transmission
+It defines the direction in which data flow between two communicating devices. There are three types of transmission modes. Read this [documentation](https://afteracademy.com/blog/what-are-the-data-transmission-modes-in-a-network) for more information on the Mode of data transmission
+

```

Comment:
In this section, you need to first introduce the three modes of transmission then direct the reader to the documentation. It is good practice to do so.